main.py

```diff
@@ -220,5 +233,7 @@ def single_argument_handler(arg):
               "\nOr type \"help\" to see all the commands.")
     elif len(argv) == 2:
         single_argument_handler(argv[1].lower())
+    elif argv[1].lower().startswith("slideshow"):
+        multiple_argument_handler(argv[1].lower(), argv[2])
     else:
         print("Only one command line argument is supported.")
```

Comment:
Since more than one command line argument is now supported, this else statement should print a different message.